.github/workflows/CI.yml

```diff
@@ -21,6 +21,12 @@ jobs:
 
       - name: Setup .NET
         run: apt install -y dotnet6
+      - name: Set execute permissions for dummy files
+        run: |
+          chmod +x src/FileConventions.Test/DummyFiles/DummyExecutable.fsx
+          chmod -x src/FileConventions.Test/DummyFiles/DummyNotExecutable.fs
```

Comment:
@realmarv rather do this in the tests please

Reply:
why?

Reply:
I'm adding execution permission to dummy files for FileConventions tests. We had a job called tests and I thought you meant I move running these tests to that job. Did you mean I do this chmods in the `Run tests to validate F# scripts` step?

Reply:
Yes because I assume that those chmod calls are needed for the tests to pass. And if a developer decides to run the tests locally, they would fail if you don't place them within the tests, because a developer doesn't run CI locally to run tests.

Reply:
Done

Reply:
This is not done yet, I still see some chmod calls in CI.yml

Reply:
Done